### PR TITLE
Updates based on Alan's feedback

### PR DESCRIPTION
**Pull request by rajinir**
Alan gave us feedback to update the deployment docs. Please review and merge

### CODE REVIEW

File: cinder/README.md [hunk 1]
```diff
@@ -214,7 +216,7 @@ unity_storage_pool_names=
 
 ## Multiple Back-ends
 
-`cinder-dellemc-unity-config.yaml` and `cinder-dellemc-vnx-config.yaml` couldn't use to configure multiple backends.
+`cinder-dellemc-unity-config.yaml` and `cinder-dellemc-vnx-config.yaml` cannot be usedto configure multiple backends.
```
Comment (shan-wan) [inline cinder/README.md:219]: typo 'usedto' , should be 'used to'

File: cinder/README.md [hunk 2]
```diff
@@ -158,18 +159,19 @@ cp /usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-unity
...
 # Cinder Dell EMC Unity backend, configured via puppet
 resource_registry:
   OS::TripleO::Services::CinderBackendDellEMCUnity: /usr/share/openstack-tripleo-heat-templates/deployment/cinder/cinder-backend-dellemc-unity-puppet.yaml
 
 parameter_defaults:
-  CinderEnableIscsiBackend: false
-  CinderEnableRbdBackend: false
-  CinderEnableNfsBackend: false
-  NovaEnableRbdBackend: false
-  GlanceBackend: file
```
Comment (shan-wan) [inline cinder/README.md:172]: why removing these?

File: cinder/README.md (hunk 1 above)
Comment (shan-wan) [reply cinder/README.md:219]: Why we cannot use them to configure multiple backends?
Actually we can use them to configure multiple backends.

File: cinder/README.md [hunk 3]
```diff
@@ -71,18 +71,19 @@ cp /usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-vnx-c
 
 Modify backend configuration in `cinder-dellemc-vnx-config.yaml`.
 
+Note: **LVM driver** is enabled by default in TripleO, you want to set the ```CinderEnableIscsiBackend``` to false in one of your environment file to turn it off.
+```yaml
```
Comment (yong-huang) [inline cinder/README.md:75]: It's better to add a blank line before and after the code block, to keep the format be consistent with other code blocks.

File: cinder/README.md [hunk 4]
```diff
@@ -158,18 +159,19 @@ cp /usr/share/openstack-tripleo-heat-templates/environments/cinder-dellemc-unity
 
 Modify backend configuration in `cinder-dellemc-unity-config.yaml`.
 
+Note: **LVM driver** is enabled by default in TripleO, you want to set the ```CinderEnableIscsiBackend``` to false in one of your environment file to turn it off.
+```yaml
```
Comment (yong-huang) [inline cinder/README.md:163]: dito

File: cinder/README.md [hunk 5]
```diff
@@ -214,9 +216,26 @@ unity_storage_pool_names=
...
 
-In order to configure multiple back-ends, we could use `ControllerExtraConfig` to set the configurations for all backends directly.
+`cinder-dellemc-unity-config.yaml` and `cinder-dellemc-vnx-config.yaml` can be used to configure both unity and vnx as multi-backends.
+
+```bash
+(undercloud) $ openstack overcloud deploy --templates \
+  -e /home/stack/templates/containers-prepare-parameter.yaml \
+  -e /home/stack/templates/custom-dellemc-container.yaml \
+  -e <other templates> \
+  -e /home/stack/templates/cinder-backend-dellemc-vnx.yaml
```
Comment (yong-huang) [inline cinder/README.md:228]: "\" is needed at EOL.

File: cinder/README.md [hunk 6]
```diff
@@ -226,6 +245,12 @@ This is an example to enable 4 backends.
 
 Create or edit `/home/stack/templates/custom-dellemc-container.yaml`.
 
+Note: **LVM driver** is enabled by default in TripleO, you want to set the ```CinderEnableIscsiBackend``` to false in one of your environment file to turn it off.
+```yaml
```
Comment (yong-huang) [inline cinder/README.md:249]: dito

File: cinder/README.md (hunk 3 above)
Comment (rajinir) [reply cinder/README.md:75]: > It's better to add a blank line before and after the code block, to keep the format be consistent with other code blocks.

Done

File: cinder/README.md (hunk 4 above)
Comment (rajinir) [reply cinder/README.md:163]: > dito

Done

File: cinder/README.md (hunk 6 above)
Comment (rajinir) [reply cinder/README.md:249]: > dito

done

File: cinder/README.md (hunk 5 above)
Comment (rajinir) [reply cinder/README.md:228]: > "" is needed at EOL.

It is there, added the \ at EOL that was missing

**Comment by rajinir** [comment]
Can we merge this PR please

**Pull request merged**